### PR TITLE
Set TomcatProperties without any Exception

### PR DESCRIPTION
**Pull request by jan-onderka**

### CODE REVIEW

File: core/src/main/groovy/noe/ews/server/tomcat/TomcatProperties.groovy [hunk 1]
```diff
@@ -2,30 +2,35 @@ package noe.ews.server.tomcat
...
-    }
-
-    if (EWS_VERSION?.trim() && !TOMCAT_MAJOR_VERSION?.trim()) {
-      throw new IllegalArgumentException("EWS_VERSION is set but the TOMCAT_MAJOR_VERSION is either null or empty")
+    Version ewsVersion = DefaultProperties.ewsVersion()
+    Version tomcatVersion = DefaultProperties.tomcatVersion()
+
+    if (tomcatVersion == null) {
+      if (ewsVersion?.getMajorVersion() == 5) {
+        TOMCAT_MAJOR_VERSION = "9"
```
Comment (honza-kasik) [inline core/src/main/groovy/noe/ews/server/tomcat/TomcatProperties.groovy]: I have no in-depth look to this problem, but doesn't this introduce similar problem again? I mean - won't this default value cause a problem? Just a note though...

Comment (jan-onderka) [reply core/src/main/groovy/noe/ews/server/tomcat/TomcatProperties.groovy]: I don't think so. It is set only for JWS 5 which has only one tomcat in version 9. So in this case is should be valid and right.

Comment (PaulLodge) [reply core/src/main/groovy/noe/ews/server/tomcat/TomcatProperties.groovy]: not to leave things: how about like this instead
```groovy
TOMCAT_MAJOR_VERSION = [3: 7, 5: 9].get(ewsVersion?.getMajorVersion(), null)
```

**Comment by honza-kasik** [comment]
To sum this up: It solves the problem when it was impossible to run tests without defined `tomcat.major.version`. @jan-onderka If you are confident, that this is the solution and it doesn't introduce any new problems which I could overlook, I can merge it now...

**Comment by honza-kasik** [comment]
@jan-onderka No response = everything is OK, merging...

**Pull request merged**